src/bundle.ts

```diff
@@ -8,11 +8,12 @@ import {
   GetRecommendations,
   NotifyRecommendationsReceived,
   GetContact,
+  PutFeedback,
 } from './index';
 
 (() => {
   const connect = (global as any).connect || {};
-  const qconnectjs = connect.qconnectjs || {};
+  const qconnectjs = connect.QConnectJS || {};
```

Comment:
this will always resolve to {} since right below you assign connect.qconnectjs in lowecase.

Reply:
let me fix it!